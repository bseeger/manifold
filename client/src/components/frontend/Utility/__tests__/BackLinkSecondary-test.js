import React from 'react';
import renderer from 'react-test-renderer';
import BackLinkSecondary from '../BackLinkSecondary';

describe("Frontend.Utility.BackLinkSecondary component", () => {

  it('renders correctly', () => {
    const component = renderer.create(
      <BackLinkSecondary
        link="test/link"
        title="test"
      />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
