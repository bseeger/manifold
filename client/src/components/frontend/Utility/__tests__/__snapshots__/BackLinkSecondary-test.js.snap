exports[`Frontend.Utility.BackLinkSecondary component renders correctly 1`] = `
<div
  className="container flush">
  <a
    className="back-link-secondary"
    onClick={[Function]}
    style={Object {}}>
    <i
      className="manicon manicon-arrow-round-left" />
    <div>
      <span
        className="back-text">
        Back to Project
      </span>
      <span
        className="project-title">
        test
      </span>
    </div>
  </a>
</div>
`;
